app/models/meetup.rb

```diff
@@ -23,9 +23,9 @@ class Meetup < ApplicationRecord
   accepts_nested_attributes_for :attachments, allow_destroy: true
   accepts_nested_attributes_for :photos, allow_destroy: true
 
-  validates :title, presence: true
-  validates :description, presence: true
-  validates :requirements, presence: true
+  validates :title, presence: true, length: { minimum: 4 }
+  validates :description, presence: true, length: { minimum: 256 }
+  validates :requirements, presence: true, length: { minimum: 4 }
```

Comment:
You should update the Meetup Model tests to take these new restrictions into account.

Also, I would set min. limits to: 10, 256 and 256 characters for title, description and requirements respectively